src/pages/node/configuration.tsx

```
@@ -33,112 +60,56 @@ function SettingsPage() {
 
-      set_strategiesString(tmp);
+      set_strategiesString(result);
+    } catch (e) {
+      console.warn('Error while counting strategies against current ticket price.', e);
     }
   }, [strategies, ticketPrice]);
 
+  useEffect(() => {
+    if (configuration) {
+      let tmp = JSON.parse(JSON.stringify(configuration));
+      tmp.hopr['strategy'] && delete tmp.hopr['strategy'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)